test/helpers/terraform-config-json/backend.json

```diff
@@ -2,19 +2,25 @@
   "provider": [
     {
       "aws": {
-        "region": "${var.region}"
+        "region": "${var.region}",
+        "__start_line__": 2,
+        "__end_line__": 7
```

Comment:
Is it possible to make it optional?
In case yes please revert all tests changes and add a new one with the feature enabled